.gitignore

```diff
@@ -6,3 +6,4 @@ VERSION
 *.rbc
 *.swp
 /.bundle
+.idea/
```

Comment:
I mean if `aa/.idea` exists, `.idea` makes sense. But if `./.idea/**`, `/.idea` is correct. `/.idea` in `.gitignore` means `./.idea/**`.